src/helics/apps/helicsWebServer.cpp

```diff
@@ -587,7 +587,9 @@ class WebSocketsession: public std::enable_shared_from_this<WebSocketsession> {
             return fail(eCode, "helics web server read");
         }
 
-        const std::string_view result{boost::asio::buffer_cast<const char*>(buffer.data()),
+        // const std::string_view result{boost::asio::buffer_cast<const char*>(buffer.data()),
+        //                              buffer.size()};
+        const std::string_view result{reinterpret_cast<const char*>(buffer.data().data()),
```

Comment:
data() has to be called twice?

Reply:
the first one extracts a mutable buffer, the second one gets the void * pointer to the raw data.  The second `.data()` is the replacement method for the buffer_cast.   I looked at that twice too, but that appears to be the way to do it in this case.